docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Overview
+type: Getting Started
+---
+
+This set of Getting Started guides is an end-to-end scenario that will walk you through all major Kyma functionalities and show its:
+- **Integration and connectivity** feature brought in by [Application Connector](https://kyma-project.io/docs/components/application-connector/). With Kyma, you can connect external applications and expose their APIs and events to the cluster.
+- **Extensibility** feature provided by [Service Catalog](https://kyma-project.io/docs/components/service-catalog/). You can use its built-in portfolio of external services in your microservices and Functions.
```

Comment:
The mapping presented in the paragraph is not so simple. We have 2 main use cases:
- extensibility
- application runtime

The main components (building blocks): application connector, service catalog, eventing, service mesh, serverless can be used in both scenarios.
 
In the extensibility use case, you build your extension and connect it to the system that you extend with Application Connector. You can host your extension in Kyma as a serverless function (very simple), microservice (more complex), or even multiple microservices (super complex extension).

In the application runtime use case, the application connector can be also used (as integration point) but it is not a mandatory component. Serverless can be used but microservice probably will work better in more complex cases. Application Gateway, Service Mesh, and Eventing probably will be the most commonly used components in that scenario. Service Catalog simplifies integration with other services (hyperscalers, SAP, or any other OSB compliant).

Reply:
@pbochynski, thanks for your input. I modified this section according to your suggestion. I haven't focused on the components this time so much (just mentioned SC but rather to explain where those services I mentioned are coming from). Take a look if it's fine now.